packages/logging/src/log-level.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * Specifies the available logging levels.
+ */
+export interface LogLevel {
```

Comment:
For TypeScript, should this be converted into an Enum?

Reply:
I think this affects custom logging and is best left as an interface. To my knowledge you add a new level to an Enum in typescript by declaring a second Enum of the same name. I think that would be difficult to do dynamically.

`LogLevel[name] = value;` for example would not work, which is how the current `addCustomLevel` function operates. Though perhaps there is an easy way of adding additional values to an Enum that I am not aware of.